### PR TITLE
remove trainings that we no longer do

### PR DESCRIPTION
**Pull request by josewenzel**

### CODE REVIEW

File: src/_data/trainings_en.yml [hunk 1]
```diff
@@ -33,22 +33,3 @@
...
-  page: /services/training/service-fabric-training
-
-- id: docker
-  name: Zero to Continous Delivery with Docker
-  author: Robert Firek
-  description: Learn to use Docker tools to enable continuous deployment, horizontal scalability & zero downtime.
-  page: /services/training/devops-training
-
-
-- id: crafting_webapps
```
Comment (threenary) [inline src/_data/trainings_en.yml:50]: I won't remove this training, we are actually re-building it for a Barcelona client. Please correct me if I am wrong @shadowman 
I imagine we should update the author @JordiCreus

**Comment by upsd** [comment]
Looks legit 👍 

Thanks for doing this.

**Review by upsd — APPROVED** [review]
LGTM

**Comment by threenary** [comment]
Why you merged this ignoring my above advice regarding the **Crafting Web Apps training**?
I would have expected some feedback from @shadowman or @JordiCreus before merging.

Unless I missed some urgent requirement to remove specifically this training I was not aware of.

File: src/_data/trainings_en.yml (hunk 1 above)
Comment (ghost) [reply src/_data/trainings_en.yml:50]: Once I've done the training we can add this again.

**Pull request merged**